app/components/account/AccountContainer.js

```diff
@@ -0,0 +1,107 @@
+// import redux, react
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Helmet from 'react-helmet';
+import { Link, browserHistory } from 'react-router';
+
+//Material UI
+import { StepsTextField, StepsRaisedButton, StepsFlatButton } from '../material-style'
+import { Paper } from 'material-ui';
+import { background, primary } from '../colors';
+import moment from 'moment';
+
+// -=-=-=-=-=-= COMPONENT =-=-=-=-=-=-
+
+export class AccountInfo extends Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {}
+  }
+
+  render() {
+
+    const {user} = this.props;
+
+    return (
+
+      <div id="account-info" className="col-xs-12" > 
+        <Helmet title="Account Info" />
```

Comment:
Yeah its in editable fields since we can add in that functionality but currently cant be edited, if I do disabled = true for each field it grays it out so you cant see the text that well